### ISSUE TITLE
Erro while connecting to local Postgresql DB

### ISSUE THREAD

**Issue by thinmanj** [Alecaddd/sequeler#61]
if you don't set the hostname neither the port the connection fails with the following message:

could not connect to server: No such file or directory
	Is the server running locally and accepting
	connections on Unix domain socket "/var/run/postgresql/.s.PGSQL.3306"?

**Comment by Alecaddd** [comment]
Ehm, Yes, that's normal. Host and port are required if not identical to the default settings.
I don't understand the issue

**Comment by thinmanj** [comment]
Oh, sorry. Didn't look for the correct port. 

Thank you.

**Comment by LinuxCoder** [comment]
The command
`createdb -p 3306 myDB`
also return the output:
`createdb: could not connect to database template1: could not connect to server: No such file or directory
	Is the server running locally and accepting
	connections on Unix domain socket "/var/run/postgresql/.s.PGSQL.3306"?`
Mmmm... I am sorry, but what a hell is that? I have created myDB base on port 3306, like in default. It should work, shouldn't it? I also tried to start postgresql using
`service posgresql start`
but after this nothing changed. Have you got any solutions?
